src/js/components/List/List.js

```diff
@@ -67,9 +70,11 @@ const List = React.forwardRef(
       focus,
       itemProps,
       pad,
+      paginate,
       primaryKey,
       secondaryKey,
-      step,
+      show: showProp,
+      step = paginate ? 10 : undefined,
```

Comment:
Thanks. I've updated to use 50. For backwards compatibility, we still need to check if user is paginating or not since previously had no default step set.

Reply:
Done.